Sources/NIOCore/EventLoopFuture+Deprecated.swift

```diff
@@ -58,44 +60,48 @@ extension EventLoopFuture {
     public func flatMapError(
         file: StaticString = #fileID,
         line: UInt = #line,
-        _ callback: @escaping (Error) -> EventLoopFuture<Value>
-    ) -> EventLoopFuture<Value> {
+        _ callback: @escaping @Sendable (Error) -> EventLoopFuture<Value>
+    ) -> EventLoopFuture<Value> where Value: Sendable {
         self.flatMapError(callback)
     }
 
+    @preconcurrency
     @inlinable
     @available(*, deprecated, message: "Please don't pass file:line:, there's no point.")
     public func flatMapResult<NewValue, SomeError: Error>(
         file: StaticString = #fileID,
         line: UInt = #line,
-        _ body: @escaping (Value) -> Result<NewValue, SomeError>
+        _ body: @escaping @Sendable (Value) -> Result<NewValue, SomeError>
```

Comment:
This requires `NewValue` to be `Sendable` but it isn't

Reply:
I don't think so. This is a simple transformation of `Value` to `Result<NewValue, SomeError>`. There is no EL hop or anything here possible since the body closure does **not** return a ELF

Reply:
Right, as the closure is `@Sendable` the `NewValue` can only have come from somewhere else if it is `Sendable`, or is produced by the closure itself. That closure executes on the EL isolation domain, so no sending is occurring here.

Reply:
Oh sorry I missed that this is synchronous :). Ignore me, completely fine as it'll be running on the EL